robot-server/tests/integration/http_api/protocols/test_v6_json_protocol.tavern.yaml

```
@@ -0,0 +1,1844 @@
+test_name: Upload, analyze, delete basic_transfer_standalone protocol.
+
+marks:
+  - usefixtures:
+      - run_server
+stages:
+  - name: Upload simple v6 protocol
+    request:
+      url: '{host:s}:{port:d}/protocols'
+      method: POST
+      files:
+        files: 'tests/integration/protocols/simpleV6.json'
```

Let's not duplicate the fixture file unnecessarily

```suggestion
        files: '../shared-data/protocol/fixtures/6/simpleV6.json'
```